src/builtins/core/year_month.rs

```diff
@@ -323,7 +323,14 @@ impl PlainYearMonth {
     }
 
     pub fn to_plain_date(&self) -> TemporalResult<PlainDate> {
-        Err(TemporalError::general("Not yet iimplemented."))
+        let iso_date = IsoDate::new_with_overflow(
```

Comment:
issue: same as the other API.

Same issue as the other comment. This one is a bit reverse though, because we need `day` rather than `year`.